### PR TITLE
Move the rules to their own page.

### PR DESCRIPTION
**Pull request by MarkBennett**
This was the simplest PR to get this content off the homepage.

I'll do a second PR to convert this to Markdown so I can grok the changes I need to do for the final prizes.

### CODE REVIEW

**Comment by MarkBennett** [comment]
Surprised by how much space these were taking on the homepage. Removing these makes it much shorter!

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <li><a href="#intro">Welcome</a></li>
         <li><a href="#join-the-fight">Join the Fight</a></li>
         <li>
-          <a href="#rules-submission-faq">Rules, Submissions &amp; FAQ</a>
```
Comment (ajyong) [inline index.html:19]: Oh!  Maybe not do this, it breaks the scrolling effect on the page.  I was picturing having a very tiny section in the main page that then has a link to the separate page.  Then, we don't break the scroll effect.

File: index.html [hunk 2]
```diff
@@ -169,344 +169,6 @@ <h3>Sponsors & Partners</h3>
     </div>
   </section>
 
-  <!-- Rules, Submissions & FAQ -->
-  <section id="rules-submission-faq" class="wrapper style2 fade-up">
-    <div class="inner">
-      <h2>Rules, Submissions &amp; FAQ</h2>
```
Comment (ajyong) [inline index.html:175]: So likely underneath here, a quick blurb to say that all this information has moved to `(add hyperlink)` a separate page.

**Review by ajyong — CHANGES_REQUESTED** [review]
~It's an unfortunate limitation of the template, but to keep the scrolling nav from breaking, we need to keep a tiny section on the main page for the rules.~

I learned relative links play nice with our template's navbar so disregard.

**Comment by ajyong** [comment]
> Surprised by how much space these were taking on the homepage. Removing these makes it much shorter!

Totally, which is why I am glad to finally see this happen.  It was tough for Marjorie and the rest of the marketing volunteers to start refinements when our screens were filled with legalese. 😆

File: index.html (hunk 1 above)
Comment (MarkBennett) [reply index.html:19]: The scrolling effect looks to be working for me, as I scroll by it just skips that section and jumps to "Links & Resources". Was there another issue you're seeing?

Can we table this for now, and I'll create a separate story to look at changing the nav? We're already doing so much work on the homepage that I'd like to wait and see how Marjorie, et al want to proceed with that work before introducing another nav element on the page somewhere else.

Comment (ajyong) [reply index.html:19]: Oh, then never mind.  Maybe it's external links that break it (we used to link outside to Google Docs).  Your changes are good then!

File: index.html (hunk 2 above)
Comment (MarkBennett) [reply index.html:175]: Let's include this in the above story, as I'd like to see how Marjorie and everyone else want to re-organize the page.

To me people already understand that links either take them to a new part of a page, or a new page entirely so I don't know if it's necessary to add an explainer.

I agree we should sort it out though, I just want to have some more context and input.

Comment (ajyong) [reply index.html:175]: Never mind, disregard since scrolling effect works.  TIL relative links are A-OK for this template.

**Review by ajyong — APPROVED** [review]

**Comment by MarkBennett** [comment]
@ajyong just made a story to track fixing the homepage nav now that the rules link is weird.

https://app.clubhouse.io/codevid19/story/469/decide-to-fit-rules-page-into-navigation-on-homepage

**Pull request merged**